commands/transform.ts

```diff
@@ -1,23 +1,22 @@
 import { join } from 'node:path'
-import execa from 'execa'
-import { bold, green } from 'picocolors'
+import type { Options } from 'jscodeshift'
+import { run as jscodeshift } from 'jscodeshift/src/Runner'
+import { bold } from 'picocolors'
 import prompts from 'prompts'
 import { TRANSFORM_OPTIONS } from '../config'
-import { getAllFiles } from '../utils/file'
 
 export function onCancel() {
   process.exit(1)
 }
 
-const jscodeshiftExecutable = require.resolve('.bin/jscodeshift')
 const transformerDirectory = join(__dirname, '../', 'transforms')
 
 // biome-ignore lint/suspicious/noExplicitAny: 'Any' is used because options can be anything.
-export async function transform(codemodName: string, source: string, options: any): Promise<void> {
+export async function transform(codemodName: string | undefined, source: string | undefined, options: any) {
```

Comment:
Parameters are set to type `undefined` because they are optional or something else?

Reply:
Because they have actually always been optional.